exercises/saddle-points/tests/saddle-points.rs

```diff
@@ -3,66 +3,65 @@ extern crate saddle_points;
 use saddle_points::*;
 
 #[test]
-fn identify_single_saddle_point() {
-    let input: Vec<Vec<u64>> = vec![vec![9, 8, 7],
-                                    vec![5, 3, 2],
-                                    vec![6, 6, 7]];
+fn test_identify_single_saddle_point() {
+    let input: &[&[u64]] = &[&[9, 8, 7],
```

Comment:
Interesting; I hadn't intended that you change the tests when I asked you to change the signature; a reference to `Vec<Vec<u64>>` should just work for a function which requires `&[&[u64]]`. 

I'm not sure whether to ask you to change this back. As of right now, I'm undecided: on the one hand, it's interesting to see that slice reference literals work at all. On the other hand, idiomatic code would be to specify the inputs as `Vec<Vec<u64>>` and then pass the reference and let the `Deref` trait do the work for you.

I'd like to solicit opinions from the other maintainers about which of these forms are better style.

Reply:
Since it is idiomatic, I would keep the `vec` then. Will be even more convinced if finding a source that tells us this, but I cannot.

We do have quite a few instances of using slices in tests in other exercises, but potentially changing them is a story for another day

Reply:
So you mean in the tests should be vectors as in the first version but I should call the function with the ```&[&[u64]]``` parameters?And the slices should be from the vectors?

Reply:
Yes: leave the function signature as it is, which expects slices of slices, and change the tests so that they allocate `Vec`s and pass a reference to the function.